pandas/tests/computation/test_eval.py

```diff
@@ -38,6 +39,23 @@
 _scalar_skip = 'in', 'not in'
 
 
+@pytest.fixture(params=[
+    pytest.mark.skipif(engine == 'numexpr' and not _USE_NUMEXPR,
+                       reason='numexpr enabled->{enabled}, '
+                              'installed->{installed}'.format(
+                            enabled=_USE_NUMEXPR,
+                            installed=_NUMEXPR_INSTALLED))(engine)
```

Comment:
sure that's reasonable.

I have always tried to see if there is a way to make a parameterized yielder

```
def create_things():
    return list(range(5))

@pytest.mark.parametrize('t', create_things())
def test_thing(t):
    ...
```

where I really want to do:

```
@pytest.fixture
def things(request):
    for in i range(5):
        yield i

@pytest.mark.parametrize('t', things):
def test_thing(t):
    ....
```
note that this is NOT a yield fixture, but something that I need a function to generate the 'things' that I want to parametrize (rather than a simple list of strings).

I think ``pytest_generate_tests`` hook would do it, but seems overly complicated.